### DISCUSSION TITLE
Others starring my repos does not show up on the Feed. Any other activity does.

### DISCUSSION THREAD

**Discussion by juhaku** [community/community#77021] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As the title states the feed does not display other people starring my repos as it used to before the overhaul. Any other activity is being shown.

Is this by desing or is there something more to it?

**Comment by iw4p** [comment]
Same here. I check them from [github.com/dashboard-feed](https://github.com/dashboard-feed)

**Comment by juhaku** [comment, marked as answer]
Seems to be fixed now? At least I am able to see the changes :thinking:
